docs/resources.md

```diff
@@ -162,6 +162,60 @@ spec:
       value: gcr.io/staging-images/kritis
 ```
 
+#### Surfacing the image digest built in a task
+
+To surface the image digest in the output of the `taskRun` the builder tool should produce this information in a [oci-layout-image](https://github.com/opencontainers/image-spec/blob/master/image-layout.md) `index.json` file. This file should be placed on a location as specified in the task definition under the resource `outputImagePath`
```

Comment:
* s/oci-layout-image/OCI Image Spec/ ?
* What happens if `index.json` contains multiple entries?
* Annotations are ignored?

Reply:
* changed `oci-layout-image` to `OCI Image Spec`
 * the index.json is per image and if there are multiple entries it should get the latest one
 * currently annotations are ignored, can you explain what we can do better?

Reply:
ignoriny is fine

Reply:
but please document that?

Reply:
added to resources.md